recipe-box-frontend/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM oven/bun:alpine
+
+# Install dependencies
+RUN apk add --no-cache nodejs
+RUN bun i -g serve
+
+# Create app folder and move local files
+RUN mkdir -p /recipe-box
+WORKDIR /recipe-box
+COPY src/main/frontend/package.json /recipe-box/
+COPY src/main/frontend/src/ /recipe-box/src/
+COPY src/main/frontend/.env /recipe-box/
+COPY src/main/frontend/public/ /recipe-box/public/
+COPY src/main/frontend/tsconfig.json /recipe-box/
+
+# Install project dependencies and build production bundle
+RUN bun i
+
+# Port to be exposed when deploying the container + env variable to use on build
+EXPOSE 80
+ENV REACT_APP_BACKEND_URL=http://localhost:8080
```

Comment:
Aici poate e posibil sa comunice containerele intre ele folosind numele lor date in docker compose in loc de localhost, cred ca gasesti niste documentatie in privinta asta

Reply:
stiu ca se fac hostname-uri pentru celelalte containere in compose, am folosit pentru baza de date, o sa schimb si pentru backend in compose doar ca in Dockerfile am lasat asa deoarece in mod normal nu stiu cum va fi folosit fisierul asta.  (poate backend-ul nu e deployed folosind docker sau e pus pe alt network asa ca am lasat localhost, oricum va trebui inlocuit cu un ip concret in momentul in care cineva da run containerului)